src/SDKs/_metadata/cognitiveservices_data-plane_QnAMaker.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/bingisbestest/azure-rest-api-specs/blob/qnav4-fixes/specification/cognitiveservices/data-plane/QnAMaker/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\dev\sdk\sdkb\src\SDKs
```

Comment:
@bingisbestest it looks like the code has been generated with a REST spec in your fork. Code must be regenerated using REST spec merged in the Azure master branch. Please regenerate the code from the correct spec and update the PR

Reply:
Yes was waiting for the change to go through. Updating.